packages/jest-types/__typetests__/jest.test.ts

```diff
@@ -98,6 +98,10 @@ expectError(jest.enableAutomock('moduleName'));
 expectType<typeof jest>(jest.isolateModules(() => {}));
 expectError(jest.isolateModules());
 
+expectType<Promise<void>>(jest.isolateModulesAsync(async () => {}));
+expectType<Promise<void>>(jest.isolateModulesAsync(() => {}));
```

Comment:
Try running type tests. This assertion is currently failing. Should be: 

```suggestion
expectError(jest.isolateModulesAsync(() => {}));
```

Reply:
mmm I did run it and it did not fail:

```
mmanciop@Micheles-MacBook-Pro jest % yarn test-types
Running one project: type-tests
 PASS   type-tests  packages/expect/__typetests__/expect.test.ts
 PASS   type-tests  packages/jest-worker/__typetests__/jest-worker.test.ts
 PASS   type-tests  packages/jest-mock/__typetests__/utility-types.test.ts
 PASS   type-tests  packages/jest-mock/__typetests__/Mocked.test.ts
 PASS   type-tests  packages/jest-types/__typetests__/each.test.ts
 PASS   type-tests  packages/jest-types/__typetests__/globals.test.ts
 PASS   type-tests  packages/jest-types/__typetests__/expect.test.ts
 PASS   type-tests  packages/jest-types/__typetests__/jest.test.ts
 PASS   type-tests  packages/jest-reporters/__typetests__/jest-reporters.test.ts
 PASS   type-tests  packages/jest-resolve/__typetests__/resolver.test.ts
 PASS   type-tests  packages/jest-runner/__typetests__/jest-runner.test.ts
 PASS   type-tests  packages/jest-mock/__typetests__/mock-functions.test.ts
 PASS   type-tests  packages/jest-snapshot/__typetests__/SnapshotResolver.test.ts
 PASS   type-tests  packages/jest-snapshot/__typetests__/matchers.test.ts
 PASS   type-tests  packages/expect-utils/__typetests__/utils.test.ts
 PASS   type-tests  packages/jest-types/__typetests__/config.test.ts
 PASS   type-tests  packages/jest-expect/__typetests__/jest-expect.test.ts
 PASS   type-tests  packages/jest-mock/__typetests__/ModuleMocker.test.ts
 PASS   type-tests  packages/jest/__typetests__/jest.test.ts

Test Suites: 19 passed, 19 total
Tests:       1283 passed, 1283 total
Snapshots:   0 total
Time:        1.951 s, estimated 2 s
Ran all test suites.
```

Reply:
The repo was not rebuild? Tests import from `@jest/globals`, not from `src`. That is why rebuilding is important after making changes in `src`.